USAGE_DATA.md

```diff
@@ -21,7 +21,7 @@ vscode-xml has opt-in telemetry collection, provided by [vscode-commons](https:/
 ## What's included in the general telemetry data
 
 Please see the
-[vscode-commons data collection information](https://github.com/redhat-developer/vscode-commons/blob/master/USAGE_DATA.md#other-extensions)
+[vscode-redhat-telemetry data collection information](https://github.com/redhat-developer/vscode-redhat-telemetry/blob/HEAD/USAGE_DATA.md#other-extensions)
```

Comment:
I think that this link should point to https://github.com/redhat-developer/vscode-redhat-telemetry/blob/HEAD/USAGE_DATA.md#usage-data-being-collected-by-red-hat-extensions.